src/PropTree.ts

```diff
@@ -1,28 +1,25 @@
 import { Dictionary } from './Dictionary';
 import { warn } from './logger';
 
-export interface IPropDesc
-{
+export interface IPropDesc {
```

Comment:
I would like to keep the previous brace style, I don't like K&R style.

Reply:
This isn't an option with prettier.